### PR TITLE
Make team and advisors editable by cms

### PR DESCRIPTION
**Pull request by ironng**
AKA (Get home page data from cms, Part 2)

This PR includes general improvements related to the team and advisor sections, as well as making them get their content from the cms. 

- `Profile` component now uses gatsby-image
- Removes extra file size versions of all profile images, except for the largest size, since gatsby-image generates what's necessary on its own
- Adds extra attributes `Icons` to make inline svg accessible to screen readers
- Updates the "See all" button to toggle the arrow direction and position depending on the collapsed/expanded state
- Renames `img` directories to `images`  (see https://github.com/keep-network/website/pull/177#discussion_r419799150)

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -29,6 +29,34 @@ collections:
...
+                widget: "list",
+                allow_add: true,
+                fields:
+                  - {label: "Name", name: "name", widget: "string"}
+                  - {label: "Title", name: "title", widget: "string"}
+                  - {
+                      label: "Profile Image",
+                      name: "image",
+                      widget: "image",
+                      hint: "Image should be at least the 2x resolution size",
```
Comment (Shadowfiend) [inline static/admin/config.yml]: Might be worth noting what this is? Is it 1024px?

File: src/components/Icons.js [hunk 2]
```diff
@@ -8,7 +8,10 @@ export const Arrow = ({ width = 59, height = 22 }) => (
     viewBox="0 0 59 22"
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
+    aria-labelledby="arrow-title"
```
Comment (Shadowfiend) [inline src/components/Icons.js:11]: 🙌

File: static/admin/config.yml [hunk 3]
```diff
@@ -57,6 +57,34 @@ collections:
...
```
Comment (Shadowfiend) [inline static/admin/config.yml]: Ditto re: what size this should be.

File: package.json [hunk 4]
```diff
@@ -32,7 +32,6 @@
     "react-dom": "^16.8.4",
     "react-helmet": "^5.2.0",
     "react-redux": "^7.2.0",
-    "react-responsive-picture": "^3.2.2",
```
Comment (Shadowfiend) [inline package.json:35]: 💥

**Comment by Shadowfiend** [comment]
Other than the size hint this is good to go.

File: static/admin/config.yml (hunk 1 above)
Comment (ironng) [reply static/admin/config.yml]: Updated!

**Pull request merged**